tests/rptest/test_suite_quick.yml

```diff
@@ -21,3 +21,4 @@ quick:
   - tests/wasm_identity_test.py
   - tests/wasm_failure_recovery_test.py
   - tests/wasm_redpanda_failure_recovery_test.py
+  - tests/partition_movement_test.py::PartitionMovementTest.test_dynamic
```

Comment:
@mmaslankaprv this test is disabled because as I increase the amount of partition movements the test fails to validate all of the offsets produced. In the test you can comment out the move_and_verify and replace it with a sleep or (or do nothing at all) to observe the background consumer/producer operating without error.